README.md

```diff
@@ -9,7 +9,8 @@
 	<a href="./LICENSE.md"><img src="https://img.shields.io/badge/license-MIT-blue.svg"></a>
 </p>
 
-<p align="center">[<a href="./docs/README.ja.md">日本語</a>]</p>
+<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a>, 
+<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```

Comment:
```suggestion
<p align="center">[<a href="https://github.com/o2sh/onefetch/docs/README.ja.md">日本語</a> | 
<a href="https://github.com/o2sh/onefetch/docs/README.fa.md">فارسی</a>]</p>
```
I think that `|` is a bit more common as a separator between languages. For example [naive-ui's README](https://github.com/TuSimple/naive-ui/blob/84f74a8f801ac93305efdc73c1e8962cfc081b48/README.md)